<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="TableWidget" name="tableWidget"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushOpen">
        <property name="text">
         <string>Открыть</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushClose">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushExit">
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1098</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Редактировать</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Разное</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_lang">
    <property name="title">
     <string>Язык</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_lang"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionInfo"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icons/icon/open.png</normaloff>:/new/icons/icon/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icons/icon/save.png</normaloff>:/new/icons/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icons/icon/close.png</normaloff>:/new/icons/icon/close.png</iconset>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icons/icon/create.png</normaloff>:/new/icons/icon/create.png</iconset>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icons/icon/edit.png</normaloff>:/new/icons/icon/edit.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать запись</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icons/icon/delete.png</normaloff>:/new/icons/icon/delete.png</iconset>
   </property>
   <property name="text">
    <string> Удалить запись</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icons/icon/info.png</normaloff>:/new/icons/icon/info.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icons/icon/exit.png</normaloff>:/new/icons/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
